pysat/_instrument.py

```diff
@@ -424,6 +456,8 @@ def __repr__(self):
                            "', clean_level='", self.clean_level,
                            "', pad={:}, orbit_info=".format(self.pad),
                            "{:}, **{:})".format(self.orbit_info, self.kwargs)])
+        # out_str = "Custom -> {:d} functions applied".format(
```

Comment:
I'm not sure if the previous repr output should be integrated here.

Reply:
We can integrate it if custom inputs were added as an optional keyword input.

Reply:
> We can integrate it if custom inputs were added as an optional keyword input.

This sounds like the way forward

Reply:
Update is in.